x-pack/plugins/lens/public/visualizations/metric/visualization.tsx

```diff
@@ -317,7 +324,8 @@ export const getMetricVisualization = ({
           supportsMoreColumns: !props.state.maxAccessor,
           filterOperations: isSupportedMetric,
           enableDimensionEditor: true,
-          supportFieldFormat: false,
+          enableFormatSelector: true,
```

Comment:
I don't think we should the max value anywhere right? If that's the case let's set this to `false`

Reply:
Good point